docs/src/adoc/index.adoc

```diff
@@ -9675,7 +9675,8 @@ link:{jdkdocs}/java.base/java/lang/reflect/Proxy.html[Java proxy^] instances.
 === GraalVM Native Image
 
 [WARNING]
-Java native is still very much under development. The current Jdbi support is only for GraalVM 25 or newer as the metadata format has changed between the last LTS (Java 21) and the current LTS (Java 25). Jdbi only supports Java 25+.
+Java native is still very much under development. The current Jdbi support is only for GraalVM 25 or newer as the metadata format has changed between the last LTS (Java 21) and the current LTS (Java 25).
+Jdbi only provides built-in native metadata for Java 25+.
```

Comment:
this duplicates the second sentence (The current Jdbi support is only for GraalVM 25 or newer)